llvm/lib/IR/BasicBlock.cpp

```diff
@@ -43,8 +41,6 @@ DbgMarker *BasicBlock::createMarker(Instruction *I) {
 }
 
 DbgMarker *BasicBlock::createMarker(InstListType::iterator It) {
-  assert(IsNewDbgInfoFormat &&
-         "Tried to create a marker in a non new debug-info block!");
```

Comment:
Are the `IsNewDbgInfoFormat` deletions (here and in the unittest) from a different patch in the series?